notebooks/community/pipelines/google_cloud_pipeline_components_cloud_natural_language_pipeline.ipynb

```diff
@@ -0,0 +1,826 @@
+{
```

Comment:
<p>use the template version</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>